index.js

```diff
@@ -635,7 +637,14 @@ function generateIndex (docIndexFile) {
 			const filename = entry.fullPath;
 			const data = matter.read(filename);
 			const title = data.data.title || pathModule.parse(filename).name;
-			const id = `${title}|${pathModule.relative('src/pages/', pathModule.dirname(filename))}`;
+
+			let result = '';
+			if (pathModule.parse(filename).name !== 'index') {
+				result = filename.replace(/(.md)$/, '');
```

Comment:
```suggestion
				result = filename.replace(/(\.md)$/, '');
```
I believe you meant this.